quantum/quantum.c

```diff
@@ -604,6 +619,13 @@ bool process_record_quantum(keyrecord_t *record) {
               PLAY_SONG(ag_swap_song);
             #endif
             break;
+          case MAGIC_SWAP_CTL_GUI:
+            keymap_config.swap_lctl_lgui = true;
```

Comment:
you can assign these at the same time:
`keymap_config.swap_lctl_lgui = keymap_config.swap_rctl_rgui = true;`